user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/SampleDetailsComponent.java

```diff
@@ -233,6 +229,7 @@ private void updateSampleGridDataProvider(ExperimentId experimentId, String filt
    * @param context the context in which the user is.
    */
   public void setContext(Context context) {
+    setSampleCount(0);
```

Comment:
Do samples depend on the experiment or the project? If they belong to the experiment this should go into setExperiment as they need to be reset everytime the experiment context changes.

Reply:
moved to setexperiment 👍